packages/ant-design/src/styles.less

```diff
@@ -0,0 +1,61 @@
+.ant-design-wallet-icon {
+    width       : 24px;
+    height      : 24px;
+    margin-right: 8px;
+    margin-left : -4px;
+}
+
+.ant-design-wallet-modal-menu {
+    border-right: 0px;
+}
+
+.ant-design-wallet-modal-menu-item {
+    margin       : 0px;
+    padding      : 0px;
+    padding-left : 0px;
+    padding-right: 0px;
+    height       : 44px;
+    line-height  : 44px;
+    box-shadow   : inset 0 -1px 0 0 rgba(255, 255, 255, 0.1);
+}
+
+.ant-design-wallet-modal-menu-button {
+    height         : 44px;
+    padding-left   : 24px;
+    display        : flex;
+    flex-direction : row-reverse;
+    justify-content: space-between;
+    align-items    : center;
+}
+
+.ant-design-wallet-modal-menu-button-icon {
+    width      : 24px;
+    height     : 24px;
+    margin-left: 8px;
+}
+
+.ant-design-wallet-multi-button-menu {
+    padding   : 0px;
+    margin-top: -44px;
+}
+
+.ant-design-wallet-multi-button-menu-wallet-name-item {
+    padding: 0px;
+}
+
+.ant-design-wallet-multi-button-menu-wallet-name-button {
```

Comment:
Would be nice to shorten some of these names --

`menu-wallet-name-item` -> `menu-item`
`menu-wallet-name-button` -> `menu-button`